README.md

```diff
@@ -317,11 +317,27 @@ For further instructions check out [configlet](https://github.com/exercism/confi
 $ GO111MODULE=off go run exercises/practice/<exercise>/.meta/gen.go
 ```
 
+**NOTE**: If you see the error `json: cannot unmarshal object into Go value of type []gen.Commit` when running the generator you probably have been rate limited by GitHub.
+Try providing a GitHub access token with the flag `-github_token="<Token>"`.
+Using the token will result in a higher rate limit.
+The token does not need any specific scopes as it is only used to fetch infos about commits.
+
 You should see that some/all of the above files have changed. Commit the changes.
 
+### Synchronizing all exercises with generators
+
+```console
+$ ./bin/run-generators <GitHub Access Token>
+```
+
+**NOTE**: If you see the error `json: cannot unmarshal object into Go value of type []gen.Commit` when running the generator you probably have been rate limited by GitHub.
+Try providing a GitHub access as the first argument of the script.
```

Comment:
:thinking: The code above already includes the token. Do you maybe mean "Make sure you provided the GitHub access token as first argument to the script as shown above."?

Reply:
I agree that your version is clearer, thanks